gasometer/src/lib.rs

```diff
@@ -409,13 +434,18 @@ pub fn call_transaction_cost(data: &[u8], access_list: &[(H160, Vec<H256>)]) ->
 		non_zero_data_len,
 		access_list_address_len,
 		access_list_storage_len,
+		authorization_list_len,
 	}
 }
 
 /// Calculate the create transaction cost.
 #[allow(clippy::naive_bytecount)]
 #[must_use]
-pub fn create_transaction_cost(data: &[u8], access_list: &[(H160, Vec<H256>)]) -> TransactionCost {
+pub fn create_transaction_cost(
+	data: &[u8],
+	access_list: &[(H160, Vec<H256>)],
+	authorization_list_len: usize,
```

Comment:
As far as I remember, we agreed that `CREATE` transactions don't have an `authorization_list`. Does it make sense to remove this parameter? Also, I can see that you pass 0 to the `record_create_transaction_cost` everywhere.

Reply:
Refactored: e85303d